src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -25,7 +25,17 @@ public partial class Activity {
     public Activity SetEndTime(DateTime endTimeUtc) {throw null;}
     public Activity Start() {throw null;}
     public void Stop() {}
-    public static Activity Current {get { throw null; } private set {} }
+    public static Activity Current 
+    {
+#if ALLOW_PARTIALLY_TRUSTED_CALLERS
```

Comment:
Thanks for the pointer, feel free to keep the #ifdef here I will voice my opinion in the related issue.